src/main/twirl/uk/gov/hmrc/play/views/layouts/main_content.scala.html

```diff
@@ -22,10 +22,9 @@
   sidebar: Html = HtmlFormat.empty,
   getHelpForm: Html = HtmlFormat.empty)
 
+<div class="header-banner-wrapper">@serviceInfo</div>
```

Comment:
Due to the `border-top: 10px solid #005ea5 ` we do need to wrap `.service-info` in a surrounding div. Otherwise there are problems with padding when moving through the breakpoints.
This seems like the simplest solution to this problem.